package.json

```diff
@@ -29,8 +30,7 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts}": [
-      "eslint . --fix",
-      "git add"
```

Comment:
`git add` is not required anymore with the new version of lint-staged we are using, that's why it has been removed 👍